tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -347,6 +357,16 @@ MakeResultPtr(const std::shared_ptr<T> &r, RLoopManager &lm, std::shared_ptr<RDF
 {
    return RResultPtr<T>(r, &lm, std::move(actionPtr));
 }
+
+template <typename T>
+std::unique_ptr<RMergeableValue<T>> GetMergeableValue(RResultPtr<T>& rptr){
+
+   if (!rptr.fActionPtr->HasRun()) rptr.TriggerRun();
+   return std::unique_ptr<RMergeableValue<T>>{
+      static_cast<RMergeableValue<T>*>(rptr.fActionPtr->GetMergeableValue())
+   };
```

Comment:
you can use `std::make_unique` (backported by ROOT in `RMakeUnique.hxx` to make this line a bit nicer (e.g. to avoid repeating the type twice)

Reply:
Thanks! didn't know that

Reply:
done

Reply:
I used std::make_unique in all the new methods in ActionHelpers.hxx.

In this function in RResultPtr.hxx though I believe it's better to leave the constructor of `std::unique_ptr` since I'm giving it the pointer to an already created RMergeableValue on the heap. Right?

Reply:
Of course since now I pass around `std::unique_ptr`s up the call chain I updated it with 
`rptr.fActionPtr->GetMergeableValue().release()`